README.md

```diff
@@ -77,7 +84,7 @@ Accordingly,
 docker-compose up -d frontend(/searchapi)
 ```
 
-Will run, from the [previous section](#default-setup), (1), (2) and (4/(5)) but skip (3), (5/(4)) and 6.
+Will run, from the [previous section](#default-setup), (1), (2) and (3/(4)) but skip (4/(3)) and (5).
```

Comment:
even if a little more verbose, but I think separating the two cases (`docker compose` up with `frontend` and `searchapi`) would make this more readable.

Reply:
thanks, I will do the change!